index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html style=" background-color:aqua;
+text-align:center;
+" >
+    <head>
+    
+    <link rel="stylesheet" href="project-1.css">
+    
+    </head>
+    <body>
+    <div class="mid" >
```

Comment:
please, use the Meaningful Tags In Html Code.

Reply:
- [ ] Kindly always remember to explain why you request a change